### DISCUSSION TITLE
Event rule action data 3.7.0

### DISCUSSION THREAD

**Discussion by arjenvri** [netbox-community/netbox#14698] [Q&A]
Does anyone know how to use the action data field in event rules of netbox 3.7.0?

Can this be used to add additional values to sent to a webhook action that can be referenced in the webhook payload body?

**Comment by jnc27** [comment]
I would also need the information how to use the action data field. Can I pass some variables, which I need for the script to run?
Thanks

**Reply by jnc27** [reply]
tried it this way

```
{
    "data": {
        "INPUT_STR": "TEST"
    }
}
```

but this did not work.

**Reply by chris240189** [reply]
the action data is probably going to change that way.
see
https://github.com/netbox-community/netbox/issues/14884
